### ISSUE TITLE
Assign custom highlighting patterns to predefined log items

### ISSUE THREAD

**Issue by bobatsar** [emilast/vscode-logfile-highlighter#168]
Is there a way to use the custom highlighting patterns and attach them to the predefined log items?

e.g.:
```
"logFileHighlighter.customPatterns": [
    {
        "pattern": "E/\\w+",
        "scope": "log.error"
    },
    {
        "pattern": "Q/\\w+",
        "scope": "log.warning"
    },
]
```

**Comment by emilast** [comment]
No, I'm afraid not. Nice idea, but with the current implementation I think it would be hard to implement.

**Comment by emilast** [comment]
@mianowill, I'm going though old issues and just saw your comment which I apparently missed before. Sorry for that. Please create a separate issue for the ESP32 log file style and supply an example and I'll see what I can do, if that would still be useful for you.

**Comment by mianowill** [comment]
@emilast totally forgot about this. Will do, ty!
